x-pack/metricbeat/module/iis/application_pool/_meta/data.json

```diff
@@ -9,7 +9,28 @@
         "application_pool": {
             "name": "test.local",
             "net_clr": {
-                "total_exceptions_thrown": 0
+                "total_exceptions_thrown": 0,
+                "finallys_per_sec": 0,
+                "exceptions_thrown_per_sec": 0,
+                "locks_and_threads": {
+                    "current_queue_length": 0,
+                    "contention_rate_per_sec": 0
+                },
+                "memory": {
+                    "gen_2_heap_size": 0,
+                    "large_object_heap_size": 0,
+                    "gen_1_heap_size": 0,
+                    "gen_1_collections": 0,
+                    "gen_0_heap_size": 0,
+                    "bytes_in_all_heaps": 0,
+                    "total_committed_bytes": 0,
```

Comment:
Can we have some actual non zero values here ?
If not all maybe for couple of them ?

Reply:
We have explored the possibility hear but the main problem here is we need to have a ASP .NET application setup to be available which we planned for future. This was discussed in the main PR.